<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1013</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="zakoncz">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>260</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ZAKONCZ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="konfiguruj_gre">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>60</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>KONFIGURUJ GRE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zapisz">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>140</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ZAPISZ</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="cpu">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>40</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>390</width>
      <height>390</height>
     </size>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sceneRect">
     <rectf>
      <x>0.000000000000000</x>
      <y>0.000000000000000</y>
      <width>300.000000000000000</width>
      <height>300.000000000000000</height>
     </rectf>
    </property>
    <property name="transformationAnchor">
     <enum>QGraphicsView::AnchorViewCenter</enum>
    </property>
    <property name="resizeAnchor">
     <enum>QGraphicsView::NoAnchor</enum>
    </property>
    <property name="rubberBandSelectionMode">
     <enum>Qt::ContainsItemShape</enum>
    </property>
    <property name="optimizationFlags">
     <set>QGraphicsView::DontAdjustForAntialiasing</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>581</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>ISOCTEUR</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; font-style:italic;&quot;&gt;POLE INFORMACYJNE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="enter_wczyt">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>360</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ENTER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="wczytaj">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>180</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>WCZYTAJ</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pole_tekst">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>330</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="player">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sceneRect">
     <rectf>
      <x>0.000000000000000</x>
      <y>0.000000000000000</y>
      <width>300.000000000000000</width>
      <height>300.000000000000000</height>
     </rectf>
    </property>
    <property name="transformationAnchor">
     <enum>QGraphicsView::AnchorViewCenter</enum>
    </property>
    <property name="resizeAnchor">
     <enum>QGraphicsView::NoAnchor</enum>
    </property>
    <property name="rubberBandSelectionMode">
     <enum>Qt::ContainsItemShape</enum>
    </property>
    <property name="optimizationFlags">
     <set>QGraphicsView::DontAdjustForAntialiasing</set>
    </property>
   </widget>
   <widget class="QPushButton" name="resetuj">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>220</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>RESETUJ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rozpocznij_gre">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>100</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ROZPOCZNIJ GRE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enter">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>330</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ENTER</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pole_tekst_wczyt">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>360</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="algorytm">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>300</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1013</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
